packages/providers/storage/chroma/src/ChromaVectorStore.ts

```diff
@@ -210,7 +210,6 @@ export class ChromaVectorStore extends BaseVectorStore {
       QueryRecordsParams
     >{
       queryEmbeddings: query.queryEmbedding ?? undefined,
-      queryTexts: query.queryStr ?? undefined,
       nResults: query.similarityTopK,
```

Comment:
A new client is requiring only one prop so I choose to pass  the `queryEmbeddings` only as we already have them at this stage.